css/base.css

```diff
@@ -1,5 +1,15 @@
 body {
   padding: 1em;
+  background-color: #2F1847;
+  /*background-color: #624763;*/
+  /*background-color: #C62E65;*/
+  /*background-color: #F9B3D1;*/
+  /*background-color: #D63AF9;*/
+  /*color: #2F1847;*/
+  /*color: #624763;*/
+  /*color: #C62E65;*/
+  color: #F9B3D1;
+  /*color: #D63AF9;*/
```

Comment:
Try and keep your code clean by leaving commented-out code out of your master branch. If you really want this to stay, you could consider placing these at the bottom of the file or something. The readability of this becomes greatly diminished if someone doesn't have code-highlighiting available. Even with it, we're left wondering why all these are here.